ci_framework/roles/tempest/tasks/configure-tempest.yml

```diff
@@ -10,7 +10,7 @@
   ansible.builtin.set_fact:
     keystone_secret_name: "{{ keystone_data.stdout | from_json | community.general.json_query('spec.secret') }}"
     keystone_passwd_select: "{{ keystone_data.stdout | from_json | community.general.json_query('spec.passwordSelectors.admin') }}"
-    keystone_api: "{{ keystone_data.stdout | from_json | community.general.json_query('status.apiEndpoint.public') }}"
+    keystone_api: "{{ keystone_data.stdout | from_json | community.general.json_query('status.apiEndpoints.public') }}"
```

Comment:
did that ever work then? Or is it a new thing? Just wanting to understand the context here.

Reply:
it always used the default for the endpoint